### DISCUSSION TITLE
群晖docker怎么搞？

### DISCUSSION THREAD

**Discussion by shit6** [HerbertHe/iptv-sources#12] [General]
大佬能否给小白出个教程，感谢

**Comment by HerbertHe** [comment]
GitHub不解决非技术问题，请Google/百度学习群晖如何使用docker，或参考其他博主的视频教程

**Reply by shit6** [reply]
docker部署完事了  但是定时任务不会了  谢谢大神解答

**Comment by HerbertHe** [comment]
本地没有群晖的测试环境

**Comment by wa19820119** [comment]
终端-新增命令启动输入crontab -e  添加 0 */2 * * * /bin/sh ./update-sources.sh 试试看吧。
